server/src/main/java/org/elasticsearch/index/mapper/GeoShapeIndexer.java

```diff
@@ -160,8 +161,9 @@ public Geometry visit(MultiPolygon multiPolygon) {
 
             @Override
             public Geometry visit(Point point) {
-                //TODO: Just remove altitude for now. We need to add normalization later
-                return new Point(point.getX(), point.getY());
+                double[] latlon = new double[]{point.getX(), point.getY()};
```

Comment:
is there a difference between `normalizePoint` and calling `normalizeLon` and `normalizeLat` directly on these values?

Reply:
It is possible to normalize longitude alone. That doesn't work for latitude thought since when you go over the pole, the longitude flips by 180 degrees.

Reply:
ah. thanks for clarifying